expression/builtin_time_vec_test.go

```diff
@@ -124,7 +124,20 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 	ast.TimestampLiteral: {},
 	ast.SubDate:          {},
 	ast.AddDate:          {},
-	ast.SubTime:          {},
+	ast.SubTime: {
+		// builtinSubTimeStringNullSig
+		{
+			retEvalType:   types.ETString,
+			childrenTypes: []types.EvalType{types.ETDatetime, types.ETDatetime},
+
+			childrenFieldTypes: []*types.FieldType{types.NewFieldType(mysql.TypeDate), types.NewFieldType(mysql.TypeDatetime)},
+
+			geners: []dataGenerator{
+				gener{defaultGener{eType: types.ETDatetime, nullRation: 0.2}},
```

Comment:
It seems we don't need this.

Reply:
Do we really need to specify a gener here? Can we use the default gener?

Reply:
So I have changed it to nil.

Reply:
Even `nil` is unnecessary. You can just remove `geners:` here.